bindings/java/src/main/com/apple/foundationdb/FutureResults.java

```diff
@@ -53,11 +60,19 @@ public RangeResult getResults() {
 		try {
 			pointerReadLock.lock();
 			if (buffer != null) {
+				if (eventKeeper != null) {
+					eventKeeper.increment(Events.RANGE_QUERY_DIRECT_BUFFER_HIT);
```

Comment:
Unlike in some other places, these events are occurring while the `pointerReadLock` is held.

Reply:
thanks for the catch